po/vi.po

```diff
@@ -19119,183 +19119,255 @@ msgid ""
 "comprehensive-rust/discussions). We would love to hear from you."
 msgstr ""
 
+# this is a translation, so the second part of the sentence need not be translated
 #: src/glossary.md
 msgid ""
 "The following is a glossary which aims to give a short definition of many "
 "Rust terms. For translations, this also serves to connect the term back to "
 "the English original."
 msgstr ""
+"Bảng từ vựng dưới đây cung cấp định nghĩa ngắn gọn của nhiều thuật ngữ Rust. "
 
 #: src/glossary.md
 msgid ""
 "allocate:  \n"
 "Dynamic memory allocation on [the heap](memory-management/stack-vs-heap.md)."
 msgstr ""
+"allocate:  \n"
+"Cấp phát bộ nhớ động trên [bộ nhớ heap](memory-management/stack-vs-heap.md)."
 
 #: src/glossary.md
 msgid ""
 "argument:  \n"
 "Information that is passed into a function or method."
 msgstr ""
+"argument:  \n"
+"Giá trị được truyền vào một hàm. Có thể được gọi là tham số."
 
 #: src/glossary.md
 msgid ""
 "Bare-metal Rust:  \n"
 "Low-level Rust development, often deployed to a system without an operating "
 "system. See [Bare-metal Rust](bare-metal.md)."
 msgstr ""
+"Bare-metal Rust:  \n"
+"Việc phát triển chương trình Rust ở để triển khai trên hệ thống không có hệ "
+"điều hành. Xem [Bare-metal Rust](bare-metal.md)."
 
 #: src/glossary.md
 msgid ""
 "block:  \n"
 "See [Blocks](control-flow/blocks.md) and _scope_."
 msgstr ""
+"block:  \n"
+"Một khối các dòng lệnh bọc trong dấu `{}`. Xem [Blocks](control-flow/blocks."
+"md) và _scope_."
 
 #: src/glossary.md
 msgid ""
 "borrow:  \n"
 "See [Borrowing](ownership/borrowing.md)."
 msgstr ""
+"borrow:  \n"
+"Một cơ chế trong Rust cho phép nhiều biến cùng truy cập một giá trị. Xem "
+"[Borrowing](ownership/borrowing.md)."
 
 #: src/glossary.md
 msgid ""
 "borrow checker:  \n"
 "The part of the Rust compiler which checks that all borrows are valid."
 msgstr ""
+"borrow checker:  \n"
+"Một bộ phận của trình biên dịch Rust kiểm tra xem tất cả các _borrow_ có hợp "
+"lệ hay không."
 
 #: src/glossary.md
 msgid ""
 "brace:  \n"
 "`{` and `}`. Also called _curly brace_, they delimit _blocks_."
 msgstr ""
+"brace:  \n"
+"`{` và `}`. Được gọi là dấu _ngoặc nhọn_, bọc quanh các _block_."
 
 #: src/glossary.md
 msgid ""
 "build:  \n"
 "The process of converting source code into executable code or a usable "
 "program."
 msgstr ""
+"build:  \n"
+"Quá trình chuyển đổi mã nguồn thành mã máy hoặc một chương trình."
 
 #: src/glossary.md
 msgid ""
 "call:  \n"
 "To invoke or execute a function or method."
 msgstr ""
+"call:  \n"
+"Gọi hoặc thực thi hàm."
 
 #: src/glossary.md
 msgid ""
 "channel:  \n"
 "Used to safely pass messages [between threads](concurrency/channels.md)."
 msgstr ""
+"channel:  \n"
+"Dùng để an toàn truyền thông điệp [giữa các _thread_](concurrency/channels."
```

Comment:
Nit: `Dùng để an toàn truyền thông điệp` -> `Dùng để truyền thông điệp một cách an toàn`.

Let's remove `_kênh_` to avoid overloading the book with new terms.